.drone.yml

```diff
@@ -60,11 +60,21 @@ pipeline:
     when:
       status: [ success, failure ]
 
+  npm-clean:
+    image: busybox
+    commands:
+      - rm -rf /drone/src/github.com/fidals/shopelectro/pull/${DRONE_PULL_REQUEST}/node_modules/*
+    volumes:
+      - /tmp/cache/drone/shopelectro/node_modules:/drone/src/github.com/fidals/shopelectro/pull/${DRONE_PULL_REQUEST}/node_modules
+    when:
+      status: [ success, failure ]
+
   docker-build:
     image: docker/compose:1.17.1
     commands:
       - cd docker
-      - docker-compose -f docker-compose.yml build --no-cache se-python
+      - docker-compose build --no-cache se-nodejs
+      - docker-compose build --no-cache se-python
```

Comment:
We may build few images inline `docker-compose build --no-cache se-nodejs se-python`

Reply:
TODO